api-dummy/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM php:7.1-fpm
```

Comment:
We can't really port the elife base images out here as they contain specific software like New Relic support